### DISCUSSION TITLE
when increased scrape_interval for slow target it is not reverted back to global value

### DISCUSSION THREAD

**Discussion by ahsanashraf918** [prometheus/prometheus#9524] [Q&A] [answered]
Hi, I observed some inconsistencies. when I increased scrape_interval for 1 of my slow target and it worked but when I revert that to default then it is not applied. can anyone help me with this
I am using file_sd_config
global scrape_interval = 15s
increased to 90s

**Comment by LeviHarrison** [comment, marked as answer]
This was resolved in https://github.com/prometheus/prometheus/issues/9549, and will be fixed in 2.31.0.
